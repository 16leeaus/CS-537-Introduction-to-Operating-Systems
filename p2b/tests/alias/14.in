alias a1 /bin/ls tests/14.desc
alias a2 /bin/cat tests/14.desc
alias a5 /usr/bin/touch tests/14.desc
a1
a2
a3
alias a3 /bin/mkdir tests-out/tmp
a3
alias
alias a4 /bin/rm -rf tests-out/tmp
a4
a5
alias
a5
unalias a5
unalias a4
alias a2 /usr/bin/touch tests/14.desc
a2
unalias a2
a2
a4
unalias a1
unalias a3
a1
a3
