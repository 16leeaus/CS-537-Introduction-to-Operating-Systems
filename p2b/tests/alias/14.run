./mysh tests/14.in | sort --ignore-case
