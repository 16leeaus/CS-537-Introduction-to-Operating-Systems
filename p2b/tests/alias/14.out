a1
a1
a1 /bin/ls tests/14.desc
a1 /bin/ls tests/14.desc
a2
a2
a2
a2 /bin/cat tests/14.desc
a2 /bin/cat tests/14.desc
a3
a3
a3
a3 /bin/mkdir tests-out/tmp
a3 /bin/mkdir tests-out/tmp
a4
a4
a4 /bin/rm -rf tests-out/tmp
a5
a5
a5 /usr/bin/touch tests/14.desc
a5 /usr/bin/touch tests/14.desc
alias
alias
alias a1 /bin/ls tests/14.desc
alias a2 /bin/cat tests/14.desc
alias a2 /usr/bin/touch tests/14.desc
alias a3 /bin/mkdir tests-out/tmp
alias a4 /bin/rm -rf tests-out/tmp
alias a5 /usr/bin/touch tests/14.desc
Creates multiple aliases and runs them, then removes those aliases and tries to run them
tests/14.desc
unalias a1
unalias a2
unalias a3
unalias a4
unalias a5
